mkdocs.yml

```diff
@@ -36,10 +36,7 @@ extra_css:
 
 plugins:
   - search
-  - progress
```

Comment:
RIP my plugin :(

oh well I don't really mind

Reply:
Docker image didn't contain it to properly build everythiung. 🤷‍♂️